### ISSUE TITLE
Custom Content Type Manager

### ISSUE THREAD

**Issue by rickysarraf** [jpSimkins/Drupal2WordPress-Plugin#27]
Hi,

First of all, thank you very much for this plugin. When following the steps, I noticed that as of Feb 2017, the mentioned module is no more available.

**Comment by jpSimkins** [comment]
Yeah, that plugin seems to have been dropped. The purpose of the plugins are only to setup your Custom post Types and/or Taxonomies. You don't need a plugin for this to work, you just need your Custom Post Types and/or Taxonomies already setup.
